### PR TITLE
Import Concourse docker updater from terraform-tomato

### PR DESCRIPTION
**Pull request by aliculPix4D**
-Import from the path: `terraform-tomato/github-automation-tools/concourse-docker-updater`
and commit 26a839ddd5c2c168fc3c741a95dd7fdf872a6945
- the imported tests are tagged as pix4d and can be executed by the following command:
`bundle exec rspec --tag pix4d`
- there remains the same number of tests as in `terraform-tomato`

### CODE REVIEW

**Comment by marco-m-pix4d** [comment]
Could you please add the PR description to the commit message?

**Comment by aliculPix4D** [comment]
> Could you please add the PR description to the commit message?

It is already there:
```
Import from the path: `terraform-tomato/github-automation-tools/concourse-docker-updater`
and commit 26a839ddd5c2c168fc3c741a95dd7fdf872a6945

* modify .gitignore, Gemfile and README.md
* import Dockerfile, update_script.rb and .rubocop.yml
* import pr_info.rb and pr_info_spec.rb
* import file_fetcher.rb, file_parser.rb, file_updater.rb and update_checker.rb from terraform-tomato
* unlike terraform version we keep the same code style as in Dependabot-core
	- no double quotes
	- package manager name is renamed back from concourse to docker
```
or you mean the second commit (regarding the tests) also?
Should I have a single commit maybe?

File: docker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,17 @@
+FROM docker.ci.pix4d.com/dependabot-ruby-base:latest AS main
```
Comment (marco-m-pix4d) [inline docker/Dockerfile]: For this PR, this is fine as-is.

But, could you please take note to discuss if this is the right place for this Dockerfile, and not something like docker/concourse/Dockerfile ?

File: docker/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+LABEL "maintainer"="integration_team@pix4d.com"
```
Comment (marco-m-pix4d) [inline docker/Dockerfile]: Please take note (for a subsequent PR) to change to platform_ci_team@pix4d.com

File: docker/README.md [hunk 3]
```diff
@@ -15,3 +15,59 @@ Docker support for [`dependabot-core`][core-repo].
    ```
 
 [core-repo]: https://github.com/dependabot/dependabot-core
+
+## Description
+This README provides details about unit testing of the conan updater dependabot code. Tests are written in `RSpec` framework https://relishapp.com/rspec. RSpec is a DSL for describing the expected behavior of Ruby code/software.
```
Comment (marco-m-pix4d) [inline docker/README.md]: Ok for this PR
Could you please take note to discuss if this file is the right place for this text ?

File: docker/Dockerfile (hunk 1 above)
Comment (aliculPix4D) [reply docker/Dockerfile]: Having in mind the epic (https://pix4dbug.atlassian.net/browse/PCI-1164) about migrating github-automation-tools to solar impulse. I already wrote some notes for future steps, and in my opinion the best place for the Dockerfiles would be to move them in `linux-image-build` repo with a trigger on this forked repository to fetch the files needed.

Comment (aliculPix4D) [reply docker/Dockerfile]: I took it here since I wanted to import everything under the import folder.

File: docker/Dockerfile (hunk 2 above)
Comment (aliculPix4D) [reply docker/Dockerfile]: Wrote a note as suggested.

File: docker/README.md (hunk 3 above)
Comment (aliculPix4D) [reply docker/README.md]: Wrote a note as suggested.

**Pull request merged**